src/Navbar/Navbar.scss

```diff
@@ -49,6 +49,13 @@
     text-decoration: none;
 }
 
+.form-container {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+}
+
```

Comment:
This is a container I added to help style an error message that is now handled in the nav bar